### DISCUSSION TITLE
Welcome to patchman Discussions!

### DISCUSSION THREAD

**Discussion by furlongm** [furlongm/patchman#490] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kraileth** [comment]
Nobody has stepped forward and tried to get the ball rolling so far - but somebody has to start, right? So here's my introduction:

I'm kraileth: sysadmin, Unix lover and Open-Source enthusiast. While I first came across patchman about a year ago, I quickly had to turn to other things before I got a chance to actually give it a try. Finally returning to the topic of more efficient patch management, I intend to seriously evaluate it.

My motivation to get in touch consists of two factors:

1.  At work I seek to replace our custom-built (and very much lacking) solution for software version and patch management with something better. Target distro for now is mostly Ubuntu.

2. Personally I'm a firm believer in platform diversity which often leads to more robust software (among other things supporting non-mainstream system as well tends to expose bugs early before they can strike hard later). In issue #457 somebody asked about BSD support and wasn't turned down immediately. Being a BSD user as well, I can share some info that might be helpful if anybody wants to give adding support for them a try (unfortunately my own Python skills are... rather limited). I can also test things or give people SSH access to temporary environments if that helps.

So let's see if I can become part of this community. I'll create two topics soon. Thanks for dedicating time to a very useful looking project!
